### PR TITLE
adding final fixes

### PR DESCRIPTION
**Pull request by iruzevic**
Final fixes before release

### CODE REVIEW

**Review by IvanGrginovInf — APPROVED** [review]

**Review by gabriel-glo — APPROVED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

File: blocks/init/src/Blocks/custom/video/video.php [hunk 1]
```diff
@@ -7,12 +7,25 @@
...
+
+$unique = Components::getUnique();
+
+?>
+
+<div class="<?php echo esc_attr($blockClass); ?>" data-id="<?php echo \esc_attr($unique); ?>">
+	<?php
+	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
+
+	echo Components::render( // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
```
Comment (dingo-d) [inline blocks/init/src/Blocks/custom/video/video.php:26]: Not valid phpcs ignore statements.

File: blocks/init/src/Blocks/custom/paragraph/paragraph.php [hunk 2]
```diff
@@ -21,9 +21,9 @@
 
 <div class="<?php echo esc_attr($blockClass); ?>" data-id="<?php echo \esc_attr($unique); ?>">
 	<?php
-	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
 
-	echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo Components::render( // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
```
Comment (dingo-d) [inline blocks/init/src/Blocks/custom/paragraph/paragraph.php:26]: Wrong phpcs ignore comment

File: blocks/init/src/Blocks/custom/lists/lists.php [hunk 3]
```diff
@@ -21,9 +21,9 @@
 
 <div class="<?php echo esc_attr($blockClass); ?>" data-id="<?php echo \esc_attr($unique); ?>">
 	<?php
-	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
 
-	echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo Components::render( // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
```
Comment (dingo-d) [inline blocks/init/src/Blocks/custom/lists/lists.php:26]: Wrong phpcs ignore comment

File: blocks/init/src/Blocks/custom/jumbotron/jumbotron.php [hunk 4]
```diff
@@ -12,7 +12,7 @@
 $manifest = Components::getManifest(__DIR__);
 $manifestBlockName = $manifest['blockName'];
 
-echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+echo Components::render( // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
```
Comment (dingo-d) [inline blocks/init/src/Blocks/custom/jumbotron/jumbotron.php:15]: Wrong phpcs ignore comment

File: blocks/init/src/Blocks/custom/image/image.php [hunk 5]
```diff
@@ -21,9 +21,9 @@
 
 <div class="<?php echo \esc_attr($blockClass); ?>" data-id="<?php echo esc_attr($unique); ?>">
 	<?php
-	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
 
-	echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo Components::render( // phpcs:ignore Eightshift.Security.CustomEscapeOutput.OutputNotEscaped
```
Comment (dingo-d) [inline blocks/init/src/Blocks/custom/image/image.php:26]: Wrong phpcs ignore comment

**Review by dingo-d — COMMENTED** [review]
There are lots of the wrong phpcs ignore comments.

Also, when writing commit messages, they should describe what the commit does. Because this will stay in the repo, and commit messages should tell a story. 

Just a note for the future, because we do have to set an example for others 😉

**Comment by iruzevic** [comment]
I have updated the coding standards to 1.3 and fixed the issues that you mentioned WordPress.Security.EscapeOutput.OutputNotEscaped

**Pull request merged**